cfme/tests/automate/custom_button/test_buttons.py

```diff
@@ -351,7 +366,44 @@ def test_custom_button_simulation():
     Bugzilla:
         1535215
     """
-    pass
+    if button_tag == "Evm":
```

Comment:
@digitronik, you are only checking for EVM and provider object type, right? LGTM.

Reply:
Yes, I don't think we need to check all object... `Simulation` mainly cover by `Automate` area. In case of `custom button` `evm` and `build` these are major type. I would like to take one one test from those.